bindings/nodejs/lib/AccountManager.ts

```diff
@@ -305,7 +305,7 @@ export class AccountManager {
     async setClientOptions(options: ClientOptions): Promise<void> {
         await this.messageHandler.sendMessage({
             cmd: 'setClientOptions',
-            payload: options,
+            payload: { clientOptions: options },
```

Comment:
Stressing me out, can we just rename options :trollface: ?

Reply:
makes sense